src/prognostic_equations/forcing/external_forcing.jl

```diff
@@ -113,7 +114,24 @@ function external_forcing_tendency!(Yₜ, Y, p, t, colidx, ::GCMForcing)
                 Lv_0 - R_v * T_0
             ) * ᶜdqtdt_sum[colidx]
         )
+    # total humidity
     @. Yₜ.c.ρq_tot[colidx] += Y.c.ρ[colidx] * ᶜdqtdt_sum[colidx]
 
+    ## subsidence -->
+    toa(f) = Spaces.level(f, Spaces.nlevels(axes(f)))  # get value at top-of-atmosphere
+    wvec = Geometry.WVector
+    ∇c = Operators.DivergenceF2C()
+    ∇(x) = begin
+        RB = Operators.RightBiasedC2F(; top = Operators.SetValue(toa(x)))
+        ∇c(wvec(RB(x)))
```

Comment:
```suggestion
        ᶜdivᵥ(wvec(RB(x)))
```
(`ᶜdivᵥ` is already defined in `src/utils/abbreviations.jl`, we can clean up subsidence for other case later.)